package io.github.orioneee.internal.processors

import io.github.orioneee.internal.domain.exceptions.AxerException

internal actual fun notifyAboutException(exception: AxerException) {}